docs/DeveloperGuide.md

```diff
@@ -133,6 +133,44 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Add feature
+GradPad allows users to add modules to their list.
+
+The following fields of a module are required (* for optional):
+* Module Code
+* Modular Credits
+* Tag*
+
+As with all operations in GradPad, the `AddCommand` class handles the execution of add operations.
+The `AddCommandParser` class helps to parse user's input before creating the correct add command.
+
+Given below is how an add operation behaves at each step of its execution.
+
+Step 1. The user types in a command string corresponding to an add operation.
+
+Step 2. This calls the `execute` method of the `LogicManager` class. The user input is passed in as a string.
+
+Step 3. `Logic.execute()` then calls the `parseCommand` method of the `gradPadParser` class to parse the string input.
+
+Step 4. `gradPadParser.parseCommand()` sees that this is an add command, and so uses the `AddCommandParser`
+class to create a corresponding `EditCommand`.
+
+Step 5. In `AddCommandParser`, the string input is first split into tokens, i.e. new module code, new tags, etc.
+
+Step 6. Then, in the same method call, a new `Module` object is created from these tokens. It now stores
```

Comment:
I think the method call wasn't specified; is it supposed to be `AddCommandParser.parse()`?

Reply:
Updated to include entire method name!